packages/neuron-ui/src/containers/Main/index.tsx

```diff
@@ -124,39 +126,52 @@ const MainContent = () => {
       <Dialog
         show={isSwitchNetworkShow}
         onCancel={onCloseSwitchNetwork}
-        onConfirm={sameUrlNetworks.length ? onSwitchNetwork : onOpenEditorDialog}
-        confirmText={sameUrlNetworks.length ? undefined : t('main.external-node-detected-dialog.add-network')}
+        onConfirm={
```

Comment:
it would be clearer if the related props can be put together

```tsx
const dialogProps = useMemo<DIalogProp>(() => {

  // using the light client
  if (isLightClientNetwork) {
    return {
      onConfirm: onCloseSwitchNetwork,
      confirmText: undefined,
      children: t('main.external-node-detected-dialog.external-node-is-light')
    }
  }
  // ...
  else if (sameUrlNetworks.length) {
  }
  // ...
  else {}

}, [isLightClientNetwork, sameUrlNetworks])


```

Reply:
Resolved